doc/build_apps/auth/jwt.rst

```diff
@@ -52,23 +46,22 @@ The CA certificate is stored with a ``set_ca_cert`` proposal:
 
 .. code-block:: bash
 
-    $ python -m ccf.proposal_generator set_ca_cert jwt_ms cacert.pem
+    $ build_proposal.sh --action set_ca_cert name jwt_ms cert_bundle @pedro_cert.pem
```

Comment:
I'm not convinced `pedro_cert.pem` is clearer than `cacert.pem`.

Reply:
`microsoft_jwt_issuer` would be clearer than `jwt_ms` on the other hand.

Reply:
Good spot - `pedro_cert.pem` was the one I had around for testing, but I agree this has a more sensible name. Have renamed the issuer too.